website/docs/r/cloud_project_kube_oidc.html.markdown

```diff
@@ -33,3 +44,17 @@ The following arguments are supported:
 * `client_id` - The OIDC client ID.
 
 * `issuer_url` - The OIDC issuer url.
+
+* `oidcUsernameClaim` - JWT claim to use as the user name. By default sub, which is expected to be a unique identifier of the end user. Admins can choose other claims, such as email or name, depending on their provider. However, claims other than email will be prefixed with the issuer URL to prevent naming clashes with other plugins.
+
+* `oidcUsernamePrefix` - Prefix prepended to username claims to prevent clashes with existing names (such as system: users). For example, the value oidc: will create usernames like oidc:jane.doe. If this field isn't set and oidcUsernameClaim is a value other than email the prefix defaults to ( Issuer URL )# where ( Issuer URL ) is the value of oidcIssuerUrl. The value - can be used to disable all prefixing.
+
+* `oidcGroupsClaim` - Array of JWT claim to use as the user's group. If the claim is present it must be an array of strings.
+
+* `oidcGroupsPrefix` - Prefix prepended to group claims to prevent clashes with existing names (such as system: groups). For example, the value oidc: will create group names like oidc:engineering and oidc:infra.
```

Comment:
```suggestion
* `oidcGroupsPrefix` - Prefix prepended to group claims to prevent clashes with existing names (such as `system:groups`). For example, the value `oidc:` will create group names like `oidc:engineering` and `oidc:infra`.
```